app/models/rejection.rb

```diff
@@ -37,6 +37,9 @@ class Rejection < ActiveRecord::Base
   validates :reasons, presence: true
   validate :validate_allowance_renews_on_date
 
+  before_save do
```

Comment:
Saving duplicates would impact stats as we aggregate per rejection reasons.
That would at least impact this CTE https://github.com/ministryofjustice/prison-visits-2/blob/master/db/views/rejection_percentage_by_days_v01.sql#L27